qiskit_ibm_runtime/runtime_program.py

```diff
@@ -115,6 +115,28 @@ def _format_common(schema: Dict) -> None:
                         + str(property_name in schema.get("required", []))
                     )
 
+        def _format_backend_requirements(schema: Dict) -> None:
+            """Add backend requirements details to `formatted`."""
+            if "input_allowed" in schema:
+                formatted.append(
+                    " " * 4 + "Input allowed: {}".format(str(schema["input_allowed"]))
+                )
+            if "min_num_qubits" in schema:
+                formatted.append(
+                    " " * 4
+                    + "Minimum number qubits: {}".format(str(schema["min_num_qubits"]))
+                )
+            if "supported_features" in schema:
+                formatted.append(
+                    " " * 4
+                    + "Supported features: {}".format(str(schema["supported_features"]))
+                )
+            for key, value in schema.items():
+                if key not in ["input_allowed", "min_num_qubits", "supported_features"]:
+                    formatted.append(
+                        " " * 4 + "{}: {}".format(sentence_case(key), str(value))
+                    )
+
```

Comment:
Input allowed and supported features will get handled by this right? Do we want to special case them?

Reply:
Not exactly, `sentence_case` converts camelCase to sentence case so the `_` would still remain.
`input_allowed` -> `Input_allowed`
`supported_features` -> `Supported_features`

I think it's better without the `_` but the properties section includes them so maybe we should just handle it here?

Reply:
Aah right we can’t use the sentence_case function. We need a new function that converts snake_case to Sentence case.